canonical-web-and-design/ubuntu.com
# Build new ubuntu advantage service description

## Done

- Build new uasd at `/legal/ubuntu-advantage-service-description`
- Build new table of contents

## QA

- View page at : https://ubuntu-com-10402.demos.haus/legal/ubuntu-advantage-service-description
- Check against [copy doc](https://docs.google.com/document/d/15URkXf2vh7bv89n-cWQPeaP2EehSNUnUCoZvGbNTYFA/edit#heading=h.eddnsno5g76l)
- Check the table of contents works as expected


## Issue / Card

Fixes [#4435](https://github.com/canonical-web-and-design/web-squad/issues/4435)
